sdk/batch/batch-rest/review/batch.api.md

```diff
@@ -134,12 +116,34 @@ export interface AzureFileShareConfigurationOutput {
     relativeMountPath: string;
 }
 
+// @public
+export type BatchAccessScope = string;
```

Comment:
Are these extensible enums?  Do we have list of known values to add for example, a `export enum KnownBatchAccessScope`? similarly for other extensible enums too.